recommender/engine/rl_agent/rl_agent.py

```diff
@@ -48,19 +77,26 @@ def _for_logged_user(self, user: User, search_data: SearchData, k: int) -> Tuple
         state_tensors = self.state_embedder(state)
 
         weights_tensor = actor_model(*state_tensors)
-        recommended_services = self.action_selector(weights_tensor, user, search_data) # TODO: jednak ids
-        recommended_services_ids = _services_to_ids(recommended_services)
+        recommended_services_ids = self.action_selector(
```

Comment:
"recommended_services_ids" -> "recommended_service_ids". (Same as rails defines methods for plural resources as this -> `User.services` and `User.service_ids`)

Reply:
This applies to every variable and method named this way, such as `get_accessed_services_ids`. This is a small change, but looks more proffesional if someone reads our code